armi/utils/directoryChangers.py

```diff
@@ -193,9 +193,22 @@ def __init__(
         DirectoryChanger.__init__(
             self, root, filesToMove, filesToRetrieve, dumpOnException
         )
-        root = root or armi.context.getFastPath()
+
+        # if no root dir is given, enforce that this tmp dir is in /.armi/
```

Comment:
I'd say this could use a little more explanation of the issues that could be caused if this check weren't performed. _Why_ is it important to check this?